pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -162,13 +210,12 @@ func (r *ReconcileCheInstallation) ensureCheNamespace(logger logr.Logger, cheIns
 	if err := r.client.Create(context.TODO(), namespace); err != nil {
 		if errors.IsAlreadyExists(err) {
 			logger.Info("Namespace for Che operator already exists", "Namespace", cheOpNamespace)
-			ns := v1.Namespace{}
+			ns := corev1.Namespace{}
 			if err := r.client.Get(context.TODO(), types.NamespacedName{Name: cheOpNamespace}, &ns); err != nil {
 				return false, err
 			}
-			if ns.Status.Phase != v1.NamespaceActive {
-				logger.Info("Namespace is not in active state", "namespace", ns.Name, "phase", ns.Status.Phase)
-				return true, nil // requeue until the namespace is active
+			if ns.Status.Phase != corev1.NamespaceActive {
+				return true, nil // requeue until namespace is active (again)
```

Comment:
I would keep the info-log there - so we know what the controller "is waiting" for
```suggestion
				logger.Info("Namespace is not in active state", "namespace", ns.Name, "phase", ns.Status.Phase)
				return true, nil // requeue until the namespace is active
```

Reply:
ok, let me revert that